src/code/z_demo.c

```diff
@@ -1,99 +1,1634 @@
+            break;
+    }
+}
+
+// Command 0x97: Set Environment Lighting
+void Cutscene_Command_SetLighting(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdEnvLighting* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        if (cmd->setting != 0x20) {
+            globalCtx->envCtx.lightSettingOverride = cmd->setting - 1;
+            globalCtx->envCtx.lightBlend = 1.0f;
+        } else {
+            globalCtx->envCtx.lightSettingOverride = 0xFF;
+        }
+    }
+}
+
+// Command 0x12C: Play Background Music or Fanfare
+void Cutscene_Command_PlaySequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        func_801A2C88(cmd->sequence - 1);
+    }
+}
+
+// Command 0x12D: Stop Background Music or Fanfare
+void Cutscene_Command_StopSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicChange* cmd) {
+    if ((csCtx->frames >= cmd->startFrame) && (cmd->endFrame >= csCtx->frames)) {
+        func_801A2D54(cmd->sequence - 1);
+    }
+}
+
+// Command 0x9C: Fade Background Music or Fanfare over duration
+void Cutscene_Command_FadeSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdMusicFade* cmd) {
+    if ((csCtx->frames == cmd->startFrame) && (csCtx->frames < cmd->endFrame)) {
+        u8 frameCount = cmd->endFrame - cmd->startFrame;
+
+        if (cmd->type == 2) {
+            Audio_QueueSeqCmd((frameCount << 0x10) | 0x110000FF);
+        } else {
+            Audio_QueueSeqCmd((frameCount << 0x10) | NA_BGM_STOP);
+        }
+    }
+}
+
+// Command 0x12E: Play Ambience music
+void Cutscene_Command_PlayAmbienceSequence(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_PlayNatureAmbienceSequence
+        // nightSeqIndex is natureAmbienceId
+        func_801A4A28(globalCtx->soundCtx.nightSeqIndex);
+    }
+}
+
+// Command 0x130:
+void func_800EAD48(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(2);
+    }
+}
+
+// Command 0x131:
+void func_800EAD7C(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    if (csCtx->frames == cmd->startFrame) {
+        // Audio_SetSfxReverbIndexExceptOcarinaBank
+        func_801A4428(1);
+    }
+}
+
+#ifdef NON_MATCHING
+// needs in-function static bss
+// audio related
+void func_800EADB0(GlobalContext* globalCtx, CutsceneContext* csCtx, CsCmdBase* cmd) {
+    static u16 seqId;
+    u8 dayMinusOne;
+
+    if (csCtx->frames == cmd->startFrame) {
+        dayMinusOne = (gSaveContext.day - 1);
+        if (dayMinusOne >= 3) {
+            dayMinusOne = 0;
+        }
+
+        switch (cmd->base) {
+            case 1:
+                // func_801A246C(SEQ_PLAYER_BGM_MAIN, TYPE_1);
+                func_801A246C(0, 1);
+                break;
```

Comment:
```suggestion
            case 1:
                // func_801A246C(SEQ_PLAYER_BGM_MAIN, TYPE_1);
                func_801A246C(SEQ_PLAYER_BGM_MAIN, 1);
                break;
```
Could you fill in the enum for the first arg in all these functions? These are in the repo now